docs/UserGuide.md

```diff
@@ -259,11 +370,14 @@ _Details coming soon ..._
 
 ## Glossary
 
-| Term          | Description                                                                                            |
-|---------------|--------------------------------------------------------------------------------------------------------|
-| **Parameter** | Information passed in as part of a command with its type identified by a prefix (e.g. `NAME`)          |
-| **Prefix**    | Characters used to identify the following parameter (e.g. `n/` is the prefix for the parameter `NAME`) |
-| **JavaFX**    | JavaFX is a set of Java graphics libraries for creating Java GUI applications                          | 
-
+| Term                               | Description                                                                                                                                                                            |
+|------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **AND search**                     | AND search finds all of the keywords.  For example, searching on Shutter AND Island returns only results that contain all two search terms.  It offers very limited results.           |
```

Comment:
Should this be `searching for  Shutter` instead?
Since this is considered a feature, could we maybe phrase the last sentence to be more positive or remove it? I feel like limited has a negative connotation, like the feature is an inconvenience, or not yet complete.

Reply:
I will remove it. I will also redo the initial sentence by using the find command as an example.